apps/files/lib/Command/ScanAppData.php

```diff
@@ -239,7 +240,7 @@ protected function showSummary($headers, $rows, OutputInterface $output) {
 	protected function formatExecTime() {
 		$secs = round($this->execTime);
 		# convert seconds into HH:MM:SS form
-		return sprintf('%02d:%02d:%02d', ($secs / 3600), ($secs / 60 % 60), $secs % 60);
+		return sprintf('%02d:%02d:%02d', intdiv($secs, 3600), intdiv($secs, 60) % 60, $secs % 60);
```

Comment:
```suggestion
		return sprintf('%02d:%02d:%02d', (int)($secs / 3600), (int)($secs / 60) % 60, $secs % 60);
```

Just converting to int is fine here too. (Similar to https://github.com/nextcloud/server/pull/32059)

This should also fix static analysis.

Reply:
I may be wrong, but this was the first thing I tried which still threw the error:
"Implicit conversion from float 0.4666666666666667 to int loses precision"

The only thing that worked was after I used intdiv. Seems odd though as I would assume statically using int should have fixed the issue, but then again I'm just a nuffy who is a beginner hobbyist coder at best!

Reply:
I just tested this and you are right. The problematic part is the modulo operator as it implicitly converts to int.

I edited my suggestion and now it should prevent any deprecation warning.